guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,22 @@
 [id="foreman-headline-features"]
 == Headline Features
 
-There are no highlights with Foreman {ProjectVersion}.
+=== IPv6 Support
+
+Deployment and operation in IPv6-only networks is now fully supported.
+
+Provisioning over IPv6 is supported on bare metal hosts.
+For compute resources, you can define the machine outside Foreman on the compute resource and then you can provision the machine as bare metal in Foreman.
+
+=== Secure Boot provisioning
+
+Provisioning with Secure Boot is now supported on bare-metal, VMware vSphere and Libvirt across multiple operating systems.
+New virtual machines with Secure Boot enabled will be created with the default keys enrolled.
+
+=== Invalidate JWT for global registration
+
+This will give users the ability to edit user permissions and invalidate JWT tokens for otherers can now invalidate their own tokens for global registration.
+Users with the `edit_users` permission can also invalidate all tokens for all users in a single action.users, invalidate self's token, and admins can invalidate selfs and other user tokens
```

Comment:
I'm not sure what happened to this sentence my suggestion, but this doesn't read well to me.

The first sentence is also just wrong in saying that it gives users the ability to edit user permissions. It doesn't. Users that have the `edit_users` permission can now invalidate tokens for other users. In my suggestion I tried to split it up into 2 clear cases: something that's true for all users and something that's true for users with a specific permission.

Also, I've never seen `self's` before and it really confuses me. I'm not a native speaker, but if I look at https://www.merriam-webster.com/dictionary/self I see no definition that fits. Shouldn't there be a possessive pronoun in this context?